src/plots/gl2d/scene2d.js

```diff
@@ -261,7 +260,7 @@ proto.updateSize = function(canvas) {
     }
 
     // make sure plots render right thing
-    if(this.redraw) this.redraw();
+    // if(this.redraw) this.redraw();
```

Comment:
Oh I forgot to remove this line... but @dfcreative @etpinard I wanted to ask about it anyway: if I leave it in, then occasionally - most noticeably when I autorange an old gl2d plot (ie pointcloud) - I get some extra tick labels: the new tick values get placed on the old axis range (if they're visible) then the correct tick labels get drawn over them without removing the obsolete ones.

Anyway I couldn't see any ill effects from removing it. OK?

Reply:
Yeah, looks like in `scene2d.plot` this

https://github.com/plotly/plotly.js/blob/8731cb5d87f305fff8386b5d8eb2f0280752393f/src/plots/gl2d/scene2d.js#L474

does the same thing. :hocho: away!

Reply:
:hocho -> https://github.com/plotly/plotly.js/pull/2437/commits/0ecaaaf8dd3fbb5d3aeb5e6aab4f4b0a0b86cfa6